Rakefile

```diff
@@ -4,3 +4,52 @@ require "rspec/core/rake_task"
 RSpec::Core::RakeTask.new(:spec)
 
 task :default => :spec
+
+namespace :openapi do
+  IBM_CLOUD_API_GEMS = %w[ibm_cloud_power ibm_cloud_vpc ibm_cloud_iam ibm_cloud_resource_controller]
+  IBM_CLOUD_OPENAPI_JSON = {
+    "ibm_cloud_power"               => "power-cloud.json",
+    "ibm_cloud_vpc"                 => "vpc.json",
+    "ibm_cloud_iam"                 => "iam-identity-token-api.json",
+    "ibm_cloud_resource_controller" => "resource-controller/resource-controller.json"
+  }
+
+  task :download_cli, [:version] do |_t, args|
+    require "net/http"
+
+    version = args[:version] || "5.0.0-beta2"
+
+    puts "Downloading openapi-generator-cli-#{version}.jar..."
+
+    uri = URI("https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/#{version}/openapi-generator-cli-#{version}.jar")
+
+    response = Net::HTTP.get_response(uri)
+    response.value
+
+    puts "Downloading openapi-generator-cli-#{version}.jar...Complete"
+
+    File.write("openapi-generator-cli", response.body)
```

Comment:
If the file already exists locally, can we just make it skip this?

Reply:
You might need to write this in binary mode?

Reply:
I'm not sure how big this gets but it might be preferable to stream the download directly into the file - https://ruby-doc.org/stdlib-2.6/libdoc/net/http/rdoc/Net/HTTP.html#class-Net::HTTP-label-Streaming+Response+Bodies

Reply:
I want to pull down updated versions but I can add the version into the pathname

Reply:
Wonder if this should be written into a special directory or somewhere so as not to clutter the root dir (also probably needs to be added to the .gitignore if not already)

Reply:
> I want to pull down updated versions but I can add the version into the pathname

Yeah, I think you can just leave the version in the file name, and then you can symlink if you want a consistent name, maybe?